tests/unittests/test_number_transform.py

```diff
@@ -0,0 +1,22 @@
+import unittest
+from tap_google_sheets.transform import transform_sheet_number_data
+
+class TestNumberTransform(unittest.TestCase):
+    """Verify that boolean values falls back as string"""
+    def test_number_transform_boolean_as_string(self):
+        """Verify that boolean values falls back as string"""
+        value = True
+        transformed_data = transform_sheet_number_data(value, sheet_title='test-sheet', col_name='test-column', col_letter='col', row_num=1, col_type='numberType')
+        self.assertIsInstance(transformed_data, str)
```

Comment:
Also, add an assertion for the value of transformed_data in all unit tests.

Reply:
Added the assertion